OpenRA.Mods.Common/Traits/Sound/AnnounceOnBuild.cs

```diff
@@ -33,7 +39,10 @@ public AnnounceOnBuild(Actor self, AnnounceOnBuildInfo info)
 
 		void INotifyBuildComplete.BuildingComplete(Actor self)
 		{
-			self.PlayVoice(info.Voice);
+			if (info.ValidStances.HasStance(self.Owner.Stances[self.World.LocalPlayer]))
```

Comment:
Actually, this crashes when `self.World.LocalPlayer` is `null`. If that is the case we might want to check `RenderPlayer`.

Reply:
will also need to handle the case where both are null.